packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -217,13 +222,21 @@ function useProcessMedia({
 
       const onUploadSuccess = ({ resource }) => {
         copyResourceData({ oldResource, resource });
-        if ('video' === resource.type && !resource.local) {
-          if (!resource.posterId) {
-            uploadVideoPoster(resource.id, resource.src);
-          }
-          if (resource.isMuted === null) {
-            updateVideoIsMuted(resource.id, resource.src);
-          }
+        if (resource.local || 'video' !== resource.type) {
```

Comment:
That means `updateBaseColor` below won't run for images.

`updateBaseColor` needs to run for images as well.

Reply:
This function is only ever passed a video resource. This check is just being overly careful.

Reply:
Ah, just realized we're in `useProcessMedia` / `trimExistingVideo` here.